README.md

```diff
@@ -1,8 +1,220 @@
+goods-movement-system-port-api
+==============================
 
-# goods-movement-system-port-api
+This API provides resources from the Goods Vehicle Movement Service (GVMS) for ports and border locations.
 
-This is a placeholder README.md for a new repository
+GVMS is used by hauliers to declare all goods movements crossing certain UK roll-on-roll-off ports.
```

Comment:
Is this necessary? is it coming from private version or new addition, just a thought that this may not be needed as this service is purely for users who manages Ports

Reply:
Can get rid (even if it was in old version, likely to have been there in error).

Reply:
<img width="894" alt="Screenshot 2023-10-11 at 11 20 28" src="https://github.com/hmrc/goods-movement-system-port-api/assets/1290092/eda3c585-936f-4227-9be9-4bb30d729b51">
That's right, from the original repo. I'm happy to remove it if no one else minds.

Reply:
gone